test/k8pshTest.cxx

```diff
@@ -0,0 +1,135 @@
+// This software is released as part of the k8psh project.
+// Portions of this software are public domain or licensed under the MIT license. (See the license file for details.)
+
+#define main k8pshMain
+#include "k8psh.cxx"
+#undef main
+
+#include <chrono>
+#include <cstdio>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <thread>
+
+#include "Test.hxx"
+
+static int runCommand(const std::string &command)
+{
+	std::cout << "Executing \"" << command << '"' << std::endl;
+
+#ifdef _WIN32
+	int exitCode = std::system(command.c_str());
+#else
+	int status = std::system(command.c_str());
```

Comment:
Use `fork()` and `execl()` (https://pubs.opengroup.org/onlinepubs/9699919799/functions/system.html) as `std::system()` blocks all other `std::system()` calls on macos (unix?).